### PR TITLE
[SYCLomatic] Update DPC++ to SYCL in code (1/2)

### PR DESCRIPTION
**Pull request by zhiweij1**
Signed-off-by: Jiang, Zhiwei <zhiwei.jiang@intel.com>

### CODE REVIEW

**Review by tomflinda — APPROVED** [review]
LGTM

**Review by ziranzha — APPROVED** [review]
LGTM

**Review by zhimingwang36 — APPROVED** [review]
LGTM

File: clang/lib/DPCT/ValidateArguments.h [hunk 1]
```diff
@@ -24,7 +24,7 @@
 /// two levels are supported currently.
 /// none:       uses helper functions from DPCT header files for memory
 ///             management migration
-/// restricted: uses API from DPC++ Explicit and Restricted Unified
+/// restricted: uses API from SYCL Explicit and Restricted Unified
```
Comment (YuriPlyakhin) [inline clang/lib/DPCT/ValidateArguments.h:27]: This text needs to be updated. it was valid, when USM was a DPC++ extension. Now it is part of SYCL 2020 standard and it doesn't have "explicit" and "restricted" levels, as far as I know.

Similar update is also needed for --usm-level=<value> option text.

Comment (zhiweij1) [reply clang/lib/DPCT/ValidateArguments.h:27]: OK, I will fix it in a separate PR.

**Pull request merged**